UM/Signal.py

```diff
@@ -189,6 +189,10 @@ def emit(self, *args, **kwargs):
     #   \param connector The signal or slot (function) to connect.
     @call_if_enabled(_traceConnect, _isTraceEnabled())
     def connect(self, connector):
+        if self._postpone_emit:
+            Logger.log("w", "Tried to connect to signal %s that is currently being postponed, this is not possible", self.__name)
```

Comment:
In this situation I think that dumping a stack trace isn't out of the question. At least then we can see what happened via the logs.